csrc/python_frontend/fusion_definition.cpp

```diff
@@ -169,6 +169,15 @@ void FusionDefinition::print(std::ostream& os) const {
   os << std::endl;
 }
 
+std::string FusionDefinition::toGraphviz(
+    IrGraphGenerator::DetailLevel detail_level) const {
+  TORCH_CHECK(
+      id().has_value(), "Cannot create graphviz until definition is complete.");
+  auto scheds = fusionCache()->queryFusionSchedules(id().value());
+  auto fusion = scheds->preschedFusion();
```

Comment:
@kevinstephano what I'd like to have is the scheduled Fusion, or the Kernel available here for different detail levels. However, I know there can be multiple of each, so the question is what is the most convenient way to either find the most recently executed one, or to allow the user to select?